### DISCUSSION TITLE
Synchronize Immich and Google Photos through Google's API

### DISCUSSION THREAD

**Discussion by simulot** [simulot/immich-go#73] [Ideas]
Instead of asking a takeout file that is more or less baldy build, why not using Google APIs to get fresh images and albums, and update Immich with them?

**Comment by hadleyrich** [comment]
There is a discussion on the one of the mmich repos about this from a few months back. I don't quite remember but there is a reason, I think there is missing info or quality when accessing through the API vs Takeout

**Reply by simulot** [reply]
This is also my findings. 

I was thinking about syncking album information. But this value is limited regarding the constraints.
